### PR TITLE
[FAB-15392] PTE supports marbles priv cc

### PR DESCRIPTION
**Pull request by dongmingh**
This CR is to
1. support marbles private cc with transient map
2. add marbles private cc to fabric-test

Signed-off-by: Dongming Hwang <dongming@ibm.com>

### CODE REVIEW

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json [hunk 1]
```diff
@@ -6,8 +6,9 @@
         "action":  "join",
         "orgName": [
             "org1",
-            "org2"
+            "org2",
+            "org3"
```
Comment (scottz64) [inline tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json:10]: ok, now I am very confused. 
Where is there a description of what is going on here, and of the tests that use these files? I need to know if you are meeting those testcase objectives, and know if the tests run successfully.

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan2-constant-q-TLS.json [hunk 2]
```diff
@@ -1,67 +0,0 @@
...
-            "fcn": "readMarble",
-            "args": ["marble"]
-        },
-        "move": {
-            "fcn": "initMarble",
-            "args": ["marble", "yellow","35","jerry"]
-        }
-    },
-    "ConnProfilePath": "ConnProfiles/test-network"
-}
```
Comment (scottz64) [inline tools/PTE/marblescc_privInputs/marblescc-priv-chan2-constant-q-TLS.json:67]: why doesn't git indicate this file is being deleted? It looks like you deleted all the lines in the file, but left the file there. Which is it?

**Review by scottz64 — CHANGES_REQUESTED** [review]

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan1-constant-d-TLS.json [hunk 3]
```diff
@@ -34,8 +34,8 @@
         "devFreq": "0"
     },
     "listOpt": {
-        "org3": ["peer1"],
-        "org4": ["peer1"]
+        "org1": ["peer1"],
+        "org2": ["peer1"]
```
[the remark this answers is not in the document]
Comment (dongmingh) [reply tools/PTE/marblescc_privInputs/marblescc-priv-chan1-constant-d-TLS.json]: ok

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan1-constant-d-TLS.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
-    "channelID": "_ch2",
+    "channelID": "_ch1",
     "chaincodeID": "marbles_cc_priv",
     "chaincodeVer": "v0",
     "logLevel": "ERROR",
-    "invokeCheck": "TRUE",
+    "invokeCheck": "FALSE",
```
[the remark this answers is not in the document]
Comment (dongmingh) [reply tools/PTE/marblescc_privInputs/marblescc-priv-chan1-constant-d-TLS.json:6]: after deleting the transaction, query will not have result

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan1-constant-t-TLS.json [hunk 5]
```diff
@@ -34,8 +34,8 @@
         "devFreq": "0"
     },
     "listOpt": {
-        "org5": ["peer1"],
-        "org6": ["peer1"]
+        "org1": ["peer1"],
+        "org2": ["peer1"]
```
[the remark this answers is not in the document]
Comment (dongmingh) [reply tools/PTE/marblescc_privInputs/marblescc-priv-chan1-constant-t-TLS.json]: ok

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json (hunk 1 above)
Comment (dongmingh) [reply tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json:10]: The purpose of the PR is to have PTE support transient map, not mean to address any testcase objectives which should handled by each testcase.

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan2-constant-q-TLS.json (hunk 2 above)
Comment (dongmingh) [reply tools/PTE/marblescc_privInputs/marblescc-priv-chan2-constant-q-TLS.json:67]: when I click "Files changed" button above it shows that this file is deleted. Not quite sure what you mean.

File: tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json (hunk 1 above)
Comment (scottz64) [reply tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json:10]: Then what is the purpose of all these files under PTE/marblescc_privInputs/ ?  If they are not functional and useful for executing a test that can be described, then why are we changing them?

**Review by scottz64 — APPROVED** [review]

**Pull request merged**